mlir/lib/Dialect/XeGPU/uArch/intel_gpu_pvc.yaml

```diff
@@ -0,0 +1,205 @@
+# This file contains the architecture-specific information for the Intel GPU codename PVC.
+
+
+---
+# The target architecture name.
+arch: intel_gpu_pvc
```

Comment:
how is this file being loaded?

Reply:
That's an older artifact, we shouldn't have yaml at all.

Reply:
Yeah, the clean up is on the way, please postpone review a bit